### PR TITLE
ENCD-5599-adjust-madqc-auditing-for-technical-replicates

### PR DESCRIPTION
**Pull request by iyoungworth**

### CODE REVIEW

File: src/encoded/audit/experiment.py [hunk 1]
```diff
@@ -1373,11 +1395,20 @@ def check_experiment_cage_rampage_standards(experiment,
...
+            mad_metrics, 'RAMPAGE (paired-end, stranded)', 0.9)
+
+        return
+
+    else:
+        yield from check_spearman(
+            mad_metrics, experiment['replication_type'],
+            0.9, 0.8, 'RAMPAGE (paired-end, stranded)')
+
+        return
```
Comment (gabdank) [inline src/encoded/audit/experiment.py]: I think the technical MAD should be limited to bulk RNA only. 
we are not planning to run and we don't have a new RAMPAGE pipeline.

File: src/encoded/audit/experiment.py [hunk 2]
```diff
@@ -1315,11 +1327,21 @@ def check_experiment_small_rna_standards(experiment,
...
+            0.9)
+
+        return
+
+    else:
+        yield from check_spearman(
+            mad_metrics, experiment['replication_type'],
+            0.9, 0.8, 'Small RNA-seq single-end pipeline')
+
+        return
```
Comment (gabdank) [inline src/encoded/audit/experiment.py]: I think the technical MAD should be limited to bulk RNA only.
we are not planning to run small RNA-seq single-end pipeline (it is the old RNA pipeline, I don't think it has to handle the tech rep MAD case)

File: src/encoded/audit/experiment.py [hunk 3]
```diff
@@ -1252,18 +1252,25 @@ def check_experiment_long_rna_standards(experiment,
...
+        mad_metrics = get_metrics(gene_quantifications,
+                                  'MadQualityMetric',
+                                  desired_assembly,
+                                  desired_annotation)
+        replicates = experiment.get('replicates')
+
+        if experiment['replication_type'] == 'unreplicated' and len(replicates) > 1:
+            yield from check_spearman_technical_replicates(
+                mad_metrics, pipeline_title, 0.9)
+
```
Comment (gabdank) [inline src/encoded/audit/experiment.py]: not a big deal, but I think you can drop the empty lines

**Review by gabdank — CHANGES_REQUESTED** [review]

File: src/encoded/audit/experiment.py (hunk 3 above)
Comment (iyoungworth) [reply src/encoded/audit/experiment.py]: Got it, I've removed the empty lines in this section

**Review by gabdank — APPROVED** [review]

**Pull request merged**